apps/profiles/tests/city_location/tests_create_city_location.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from django.contrib.gis.geos import Point
+from rest_framework.reverse import reverse
+
+from apps.profiles.models import CityLocation
+from apps.profiles.tests.city_location.constants import CITY_LIST_URL
+from apps.profiles.tests.utils import get_tokens
+
+
+@pytest.mark.django_db
+def test_create_city_valid(api_client,
+                           user, city_location_data):
+    token = get_tokens(user)
+    api_client.credentials(HTTP_AUTHORIZATION="Bearer " + token)
```

Comment:
Maybe we can make a separate fixture to get user token and set it into headers? We could possibly call it: authenticate or smth similar

Reply:
Add separated fixture into file